src/scss/theme/banners/_top-banner--image-and-text.scss

```diff
@@ -13,6 +13,14 @@
         &.su-hero__card {
           @include saa-banner--bright-red-card-common;
           max-width: 50%;
```

Comment:
Please change this 50% width setting to be inside a min-width media query for larger screens. thanks!